test/parallel/test-promises-unhandled-rejections.js

```diff
@@ -293,7 +293,7 @@ asyncTest('While inside setImmediate, catching a rejected promise derived ' +
   });
 });
 
-// State adapation tests
+// State adaption tests
```

Comment:
I don't really understand what this comment is saying (before or after). :( I think the tests describe what they do well enough.
 
(If we are to keep it, I would prefer to use the more common version "adaptation".)

Reply:
I don't fully understand that test either, I went with "adaptation" for now. If someone knows what's going on here, we might actually remove it.